### PR TITLE
Version Update Changes

### PR DESCRIPTION
**Pull request by SamuelGabriel**
### Change Description

**Brief** (a few bullet points describing your changes, use full sentences and try to link lines in the code whenever needed)

I needed to change the version to be able to upload the package to Pypi.
I ran into some errors and added printing for these s.t. users can send us better bug reports in that case (not a perfect solution at all, as it only is triggered for this one error.)

### Standard Qs (leave questions that do not apply blank)
**If you broke behavior: Please describe what behavior you broke and how you inform people to not get stuck trying to use the old behavior.**
none

**If you used new dependencies: Did you add them to `requirements.txt`?**
no

**Who did you ping on Mattermost to review your PR? Please ping that person again whenever you are ready for another review.**
@davidotte @liam-sbhoo 

---
*Please do not mark comments/conversations as resolved unless you are the assigned reviewer. This helps maintain clarity during the review process.*

### CODE REVIEW

**Review by davidotte — APPROVED** [review]
Looks good

File: tabpfn_client/client.py [hunk 1]
```diff
@@ -196,7 +196,10 @@ def try_connection(self) -> bool:
             if response.status_code == 200:
                 found_valid_connection = True
 
-        except httpx.ConnectError:
+        except httpx.ConnectError as e:
+            logger.error(f"Failed to connect to the server with error: {e}")
+            import traceback
```
Comment (liam-sbhoo) [inline tabpfn_client/client.py:201]: I'd move the import to the file for better readability :)

**Review by liam-sbhoo — APPROVED** [review]
Thanks! I only had one very minor comment.

File: tabpfn_client/client.py (hunk 1 above)
Comment (SamuelGabriel) [reply tabpfn_client/client.py:201]: ok i will do this in the new pr

**Pull request merged**